Ignore the default include dir -I/usr/include

Upstream-status: Inappropriate [embedded specific]

The output from `icu-config --cppflags-searchpath` is "-I/usr/include" and will
lead to host contamination. Just ignore it since that's the default path.

Signed-off-by: Jesse Zhang <sen.zhang@windriver.com>

--- xerces-c-3.1.1/m4/xerces_icu_prefix.m4.orig	2012-10-16 16:04:26.300582775 +0800
+++ xerces-c-3.1.1/m4/xerces_icu_prefix.m4	2012-10-16 16:06:41.659593078 +0800
@@ -38,7 +38,7 @@
                   # can use them uniformly whether they are empty or not.
                   #
                   if test -n "$icu_config"; then
-                    icu_flags=`$icu_config --cppflags-searchpath`
+                    icu_flags=`$icu_config --cppflags-searchpath | sed s:-I/usr/include::`
 		    icu_libs=`$icu_config --ldflags`
 		    icu_bin=`$icu_config --bindir`
 		    icu_sbin=`$icu_config --sbindir`
